pkg/template/v2/pem.go

```diff
@@ -56,6 +56,57 @@ func filterPEM(pemType, input string) (string, error) {
 	return string(blocks), nil
 }
 
+// getCertificates - returns a list of certs from a pem which may contain multiple certs.
+func getCertificates(input string) ([]string, error) {
+	certs := make([]string, 0)
+
+	data := []byte(input)
+	var blocks []byte
+	var block *pem.Block
+	var rest []byte
+	for {
+		block, rest = pem.Decode(data)
+		data = rest
+
+		if block == nil {
```

Comment:
Interestingly enough this will ignore junk in the cert after it found at least one working cert. Which might be okay 🤔